app/templates/components/project-geometries/types/development-site.hbs

```diff
@@ -21,21 +21,21 @@
           </span>
           polygon tool to add a new shape
         </li>
-        <li>Complete a new shape by clicking its final vertex a second time</li>
+        <li>Complete a new shape by clicking its final corner a second time</li>
       </ul>
 
       <p class="text-weight-bold small-margin-bottom">Editing existing shapes:</p>
       <ul>
-        <li>Edit an existing shape by selecting it and dragging its vertices</li>
-        <li>Add a vertex by clicking the midpoint of a line</li>
+        <li>Edit an existing shape by selecting it and dragging its corners</li>
```

Comment:
Maybe all the "corners" should be "points" so that we're not using corner/point interchangeably to mean the same thing. And as pointed out in other comments, there are issues with saying "corner."